core/src/main/java/org/elasticsearch/client/transport/TransportClientNodesService.java

```diff
@@ -431,11 +431,11 @@ public void run() {
                                     // if its one of the actual nodes we will talk to, not to listed nodes, fully connect
                                     if (nodes.contains(listedNode)) {
                                         logger.trace("connecting to cluster node [{}]", listedNode);
-                                        transportService.connectToNode(listedNode);
+                                        transportService.connectToNodeLight(listedNode);
                                     } else {
                                         // its a listed node, light connect to it...
                                         logger.trace("connecting to listed node (light) [{}]", listedNode);
-                                        transportService.connectToNodeLight(listedNode, pingTimeout, !ignoreClusterName);
+                                        transportService.connectToNodeLight(listedNode);
```

Comment:
This one is correct as-is. I wish I had an understanding of what I did to cause the other two flips (mostly to avoid whatever it was in the future).